cypress/integration/E2E_tests/NavBarTests/navBar.spec.js

```diff
@@ -3,6 +3,14 @@ describe('NavBar can', () => {
         cy.login();
         cy.wait(2000);
     })
+    //This test mightg need some work as it's testing something super specific
+    it('Redirect you to a Yahoo page',()=>{
+        cy.get("[data-cy=autocomplete-search-bar]").should("be.visible");
+        cy.get("[data-cy=autocomplete-search-bar]").type("A");
+        cy.get("[data-cy=autocomplete-list-content]").should("be.visible");
+        cy.get("[data-cy=autocomplete-list-item]").should("be.visible");
+        cy.get("[data-cy=autocomplete-list-item]").contains("AAPL").click();
```

Comment:
Is there a way to confirm that a new tab has been opened (technically, if I understand the test correctly, if it wasn't redirected, it would still pass)?

Reply:
Yes that is correct, however I did some research and the only way to validate that the link opened would be to have a separate cypress instance validate it. I will try to find a way to see if I can validate whether its the correct link by validating what the function is returning.

Reply:
Good enough for me, let me know if you find a way to validate the URL.

Reply:
The redirect method is no longer working and I don't know why